### PR TITLE
MTDSA-25019: reinstate ITs in v4

### PR DESCRIPTION
**Pull request by LouiseWong98**

### CODE REVIEW

File: it/v4/endpoints/bfLoss/amend/AmendBFLossControllerISpec.scala [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+import play.api.test.Helpers.AUTHORIZATION
+import support.V4IntegrationBaseSpec
+import support.stubs.{AuditStub, AuthStub, DownstreamStub, MtdIdLookupStub}
+
+class AmendBFLossControllerISpec extends V4IntegrationBaseSpec {
+
+  val lossAmount = 2345.67
+
+  val downstreamResponseJson: JsValue = Json.parse(s"""
+                                                      |{
```
Comment (mattstephens-hmrc) [inline it/v4/endpoints/bfLoss/amend/AmendBFLossControllerISpec.scala:36]: I guess these are just being reinstated, but while you're there, can you move this block over to the left so it's in line with the requestJson one below it?

+ any others that are like that

File: it/v4/endpoints/bfLoss/delete/DeleteBFLossControllerISpec.scala [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        }
+      }
+
+      validationErrorTest("BADNINO", "AAZZ1234567890a", BAD_REQUEST, NinoFormatError)
+      validationErrorTest("AA123456A", "BADLOSSID", BAD_REQUEST, LossIdFormatError)
+    }
+
+  }
+
+}
```
Comment (mattstephens-hmrc) [inline it/v4/endpoints/bfLoss/delete/DeleteBFLossControllerISpec.scala:133]: Could you hit Reformat on this (probably the other files too? Can do it at the folder-level).

File: it/v4/endpoints/bfLoss/create/CreateBFLossControllerISpec.scala [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+import com.github.tomakehurst.wiremock.stubbing.StubMapping
+import play.api.http.HeaderNames.ACCEPT
+import play.api.http.Status._
+import play.api.libs.json._
+import play.api.libs.ws.{WSRequest, WSResponse}
+import play.api.test.Helpers.AUTHORIZATION
+import support.V4IntegrationBaseSpec
+import support.stubs.{AuditStub, AuthStub, DownstreamStub, MtdIdLookupStub}
+
+class CreateBFLossControllerISpec extends V4IntegrationBaseSpec with JsonErrorValidators {
```
Comment (mattstephens-hmrc) [inline it/v4/endpoints/bfLoss/create/CreateBFLossControllerISpec.scala:30]: Just noticed, the existing v5 ITs are also extending V4IntegrationBaseSpec. Could be worth updating them while you're there.

That said, these shouldn't really have "V4, V5 etc"-IntegrationBaseSpecs, just the one IntegrationBaseSpec, to be in line with the other ITs.   Up to you if you want to fix these now, or create a separate story to do so.

File: it/v4/endpoints/bfLoss/amend/AmendBFLossControllerISpec.scala (hunk 1 above)
Comment (LouiseWong98) [reply it/v4/endpoints/bfLoss/amend/AmendBFLossControllerISpec.scala]: updated!

**Review by mattstephens-hmrc — APPROVED** [review]

**Pull request merged**